### DISCUSSION TITLE
Resteasy has local cache space issues for sending large files

### DISCUSSION THREAD

**Discussion by CongminYang** [resteasy/resteasy#3261] [Q&A] [answered]
When using Resteasy to send large files, such as hundreds of gigabytes, the local space is insufficient. How to solve the problem of sending large files?

**Comment by liweinan** [comment, marked as answer]
@CongminYang I assume you are using RESTEasy Client to send large files. Have you tried with the chunked mode:

- https://docs.jboss.org/resteasy/docs/6.1.0.Final/userguide/html/RESTEasy_Client_Framework.html

In above document it says:

> One RESTEasy extension to the client API is the ability to specify that requests should be sent in "chunked" transfer mode. There are two ways of doing that.

Maybe this is useful to you.
